light_node/src/configuration.rs

```diff
@@ -939,6 +939,13 @@ impl Environment {
                     |value| {
                         value
                             .parse::<KeyValueStoreBackend>()
+                            .map( |v| {
```

Comment:
@bkontur not sure if that can be handled in a better way to still fit into that nice refactoring you just made ...

Reply:
> 
> 
> @bkontur not sure if that can be handled in a better way to still fit into that nice refactoring you just made ...

I guess, this is the correct place to handle this